.github/pr-labeler.yml

```diff
@@ -0,0 +1,4 @@
+enhancement: ['feature/*', 'feat/*', 'enhancement/*']
```

Comment:
I would make an other label for feature. Like this in the changelog you will not see the new feature mixed with the refactors for instance.

Reply:
done